.github/workflows/event-merge-to-queue.yml

```diff
@@ -15,87 +15,69 @@ concurrency:
     if: needs.docs-only.outputs.only-docs-changed == 'false'
     strategy:
       matrix:
-        test-config: # configurations for both standalone and coordinator
-          ${{ fromJson(needs.print-configurations.outputs.test-configurations) }}
-        include: # spacial configurations
-          ${{ fromJson(needs.print-configurations.outputs.include-configurations) }}
+        redis-ref:
+          - ${{ needs.get-latest-redis-tag.outputs.tag }}
+          - ${{ needs.get-latest-7-0-redis-tag.outputs.tag }}
+          - ${{ needs.get-latest-6-2-redis-tag.outputs.tag }}
+          - ${{ needs.get-latest-6-0-redis-tag.outputs.tag }}
+    uses: ./.github/workflows/flow-linux-platforms.yml
+    secrets: inherit
+    with:
+      redis-ref: ${{ matrix.redis-ref }}
+      architecture: x86_64
 
+  test-linux-aarch64:
+    needs: [docs-only, get-latest-redis-tag]
+    if: needs.docs-only.outputs.only-docs-changed == 'false'
     uses: ./.github/workflows/flow-linux-platforms.yml
     secrets: inherit
     with:
       redis-ref: ${{ needs.get-latest-redis-tag.outputs.tag }}
-      test-config: CONFIG=${{ matrix.test-config }}
-      standalone:  ${{ matrix.standalone  != 'false' }} # Set the default value to `true` if standalone is absent
-      coordinator: ${{ matrix.coordinator != 'false' }} # Set the default value to `true` if coordinator is absent
+      test-config: QUICK=1
+      architecture: aarch64
 
   test-macos:
-    needs: [print-configurations, docs-only, get-latest-redis-tag]
+    needs: [docs-only, get-latest-redis-tag]
     if: needs.docs-only.outputs.only-docs-changed == 'false'
     strategy:
-      fail-fast: false
       matrix:
-        dividor: [true, false]
-        # configurations for both standalone and coordinator
-        test-config: ${{ fromJson(needs.print-configurations.outputs.test-configurations) }}
-        # spacial configurations
-        include: ${{ fromJson(needs.print-configurations.outputs.include-configurations) }}
-
+        mode: [standalone, coordinator]
     uses: ./.github/workflows/flow-macos.yml
     secrets: inherit
     with:
       redis-ref: ${{ needs.get-latest-redis-tag.outputs.tag }}
-      test-config: CONFIG=${{ matrix.test-config }}
-      # Set the default value to `true` if value is absent
-      # Using the dividor, only one of the two will be true, and we split the run into two jobs.
-      # If the configuration was ment to only one of the modes, we will get for example: (true, false), (false, false)
-      # and if both modes are false, the job will be skipped (see task-test.yml)
-      standalone:  ${{ matrix.standalone  != 'false' &&  matrix.dividor }}
-      coordinator: ${{ matrix.coordinator != 'false' && !matrix.dividor }}
+      test-config: QUICK=1
+      standalone:  ${{ matrix.mode == 'standalone'  }}
+      coordinator: ${{ matrix.mode == 'coordinator' }}
```

Comment:
`standalone` and `coordinator` are not complementary, specifically on the Mac flow here we want to split the `task-test.yml` flow so it will run faster